magento/magento2
# Remove VarienForm as it no longer appears to be included or used.

### Description
This code no longer appears to be included from anywhere within the core, 
I see resolvers that mention the directory but again no direct calls that I can see:
- setup/src/Magento/Setup/Module/I18n/Dictionary/Options/Resolver.php
- setup/src/Magento/Setup/Test/Unit/Module/I18n/Dictionary/Options/ResolverTest.php

### Manual testing scenarios
1. Ensure there isn't any use of VarienForm, RegionUpdater and ZipUpdater from form.js

### Contribution checklist
